src/backend/joanie/core/templates/admin/index.html

```diff
@@ -0,0 +1,8 @@
+{% extends "admin/index.html" %}
+
+{% block content %}
+  <h2>
+    <a href={% url "sentry-decrypt" %}>Decrypt sentry context</a>
+  </h2>
+  {{block.super}}
+{% endblock %}
```

Comment:
We could override the base.html template instead. WDYT? 

```
{% extends "admin/base.html" %}

{% block userlinks %}
    <a href={% url "sentry-decrypt" %}>{% trans "Sentry tools" %}</a> /
  {{block.super}}
{% endblock %}
```

<img width="641" alt="image" src="https://github.com/openfun/joanie/assets/9265241/6a57d6c7-e31f-47ce-9fa1-d7254cf52bdb">